src/Kestrel.Core/Internal/Http/Http1Connection.cs

```diff
@@ -354,29 +355,42 @@ private static unsafe string GetUtf8String(Span<byte> path)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private void EnsureHostHeaderExists()
```

Comment:
Skip this method, I'm workong on some changes here that will conflict.

Reply:
You can fix the conflicts...

Reply:
It's not that simple, my changes will negate these.

Reply:
Is your change going to be done in the next few days?

Reply:
> It's not that simple, my changes will negate these.

Just overwrite it? Isn't a conflict (except to git) if you entirely replace it; and it give you a perf base line to measure against 😉 

Want me to drop this change?

Reply:
Yep, drop the change.

Reply:
Reverted